src/idd/idd.utils.js

```diff
@@ -235,6 +235,39 @@ InteractiveDataDisplay.Utils =
             return { minx: minx, maxx: maxx, miny: miny, maxy: maxy };
         },
 
+        enumPlots: function (masterPlot, plots) {
```

Comment:
@ElenaPochernina Two things are not good: 
1) Why do I need to pass an array as input when I call Utils.enumPlots?
2) This function now orders elements on each recursive call instead of ordering it once.

Therefore I suggest the following:
1) enumPlots: function(plot) which then uses plot.master, creates an empty array, declares 
a local function "var enumRec (p, plots) = ..." which just adds the plots to the given array;
2) When enumRec finishes and returns the final array, you do ordering (once).